content/id/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -0,0 +1,449 @@
+*status condition* yang ditetapkan oleh Kubernetes pada HorizontalPodAutoscaler. _Status condition_ ini menunjukkan apakah HorizontalPodAutoscaler dapat menyekalakan atau tidak, dan apakah saat ini dibatasi atau tidak.
+
+Kondisi muncul di _field_ `status.conditions`. Untuk melihat kondisi yang memengaruhi HorizontalPodAutoscaler, kita bisa menggunakan `kubectl description hpa`:
+
+```shell
+kubectl describe hpa cm-test
+```
+```shell
+Name:                           cm-test
+Namespace:                      prom
+Labels:                         <none>
+Annotations:                    <none>
+CreationTimestamp:              Fri, 16 Jun 2017 18:09:22 +0000
+Reference:                      ReplicationController/cm-test
+Metrics:                        ( current / target )
+  "http_requests" on pods:      66m / 500m
+Min replicas:                   1
+Max replicas:                   4
+ReplicationController pods:     1 current / 1 desired
+Conditions:
+  Type                  Status  Reason                  Message
+  ----                  ------  ------                  -------
+  AbleToScale           True    ReadyForNewScale        the last scale time was sufficiently old as to warrant a new scale
+  ScalingActive         True    ValidMetricFound        the HPA was able to successfully calculate a replica count from pods metric http_requests
+  ScalingLimited        False   DesiredWithinRange      the desired replica count is within the acceptable range
+Events:
+```
+
+Untuk HorizontalPodAutoscaler ini, kita dapat melihat beberapa kondisi yang menandakan dalam keadaan sehat. Yang pertama, `AbleToScale`, menunjukkan apakah HPA dapat mengambil dan memperbarui skala atau tidak, serta apakah kondisi terkait backoff akan mencegah penyekalaan atau tidak. Yang kedua, `ScalingActive`, menunjukkan apakah HPA diaktifkan atau tidak (yaitu jumlah replika target bukan nol) dan mampu menghitung skala yang diinginkan. Jika `False`, biasanya menunjukkan masalah dengan
+pengambilan metrik. Terakhir, kondisi terakhir, `ScalingLimited`, menunjukkan bahwa skala yang diinginkan telah dibatasi oleh maksimum atau minimum HorizontalPodAutoscaler.  Ini adalah indikasi bahwa Anda mungkin ingin menaikkan atau menurunkan batasan jumlah replika minimum atau maksimum pada HorizontalPodAutoscaler kamu.
+
+## Lampiran: Kuantitas
+
+Semua metrik di HorizontalPodAutoscaler dan Metrics API ditentukan menggunakan notasi bilangan bulat khusus yang dikenal di Kubernetes sebagai *kuantitas*. Misalnya, kuantitas `10500m` akan ditulis sebagai `10.5` dalam notasi desimal. Metrics API akan menampilkan bilangan bulat tanpa sufiks jika memungkinkan, dan secara umum akan mengembalikan kuantitas dalam satuan mili.  Ini berarti Anda mungkin melihat nilai metrik Anda berfluktuasi antara `1` dan` 1500m`, atau `1` dan` 1,5` ketika ditulis dalam notasi desimal. Lihat
+[entri glosarium tentang kuantitas](/docs/reference/glossary?core-object=true#term-quantity) untuk informasi lebih lanjut.
```

Comment:
this can link to our indonesian glossary once ready

Reply:
Actually it's my mistakes because I translates from older pages of english documentation file.
The current english documentation file has glossary tooltip to the quantity.
I already add the `kuantitas.md` and use the current english documentation translation.
Thank you for noticing that.